from src.excepciones.ErrorAplicacion import ErrorAplicacion
class ExceptionTypeB(ErrorAplicacion):
    def __init__(self):
        pass